db/migrate/20170413152832_add_profile_phone_confirmed.rb

```diff
@@ -0,0 +1,5 @@
+class AddProfilePhoneConfirmed < ActiveRecord::Migration
+  def change
+    add_column :profiles, :phone_confirmed, :boolean, default: false, null: false
```

Comment:
IME the index is only required if you intend to search or sort on a column. Otherwise it adds extraneous db overhead to keep the index updated. We only ever check the value of this flag *after* we've searched by `user_id`.